syntax = "proto3";
package applicationVulnerabilityServiceIf;

/*
 * VulenarabilityScoreService takes FileInfo and returns the max CVSS
 */

service VulnerabilityScoreService {
    rpc getVulnerabilityScore(FileInfo) returns (CVSS) {}
}


/*
 * FileInfo can be any of file hash or file name (as of now)
 */
message FileInfo {
    string file_hash = 1;
}

/*
 * CVSS is a float value
 */

message CVSS {
    float cvss = 1;
}

